### PR TITLE
acl in args

### LINKED ISSUE #2: stick to new s3 bucket defaults policies ?
https://github.com/apancutt/deploy-aws-s3-cloudfront/blob/3857dc445fbb63cc7c3fb8212b9555fe1d6e5bd3/bin/deploy-aws-s3-cloudfront.js#L293

New created buckets are not public and public acl are forbiddens.
Public access are given through the cloudfront.
I think this is a good pattern and it should be possible to add this option in the cmd.

### PR DESCRIPTION
**Pull request by jclappiway**
should fix #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ If you are relying on credentials stored in `~/.aws/credentials` you can use the
 
 The name of the S3 bucket to sync to.
 
+#### `--acl public`
```
Comment (apancutt) [inline README.md]:
```suggestion
#### `--acl <canned-acl>`
```

File: README.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ If you are relying on credentials stored in `~/.aws/credentials` you can use the
...
+
+Add public acl.
```
Comment (apancutt) [inline README.md]:
```suggestion
A canned ACL string. See https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property for accepted values.
```

File: bin/deploy-aws-s3-cloudfront.js [hunk 3]
```diff
@@ -290,9 +294,12 @@ function deploy(uploads, deletes) {
 
         const defaults = {
           Bucket: argv.bucket,
-          ACL: 'public-read',
         };
-
+        
+        if(argv.acl === 'public'){
```
Comment (apancutt) [inline bin/deploy-aws-s3-cloudfront.js]:
```suggestion
        if (argv.acl) {
```

File: bin/deploy-aws-s3-cloudfront.js [hunk 4]
```diff
@@ -290,9 +294,12 @@ function deploy(uploads, deletes) {
...
+          defaults.ACL= 'public-read'
```
Comment (apancutt) [inline bin/deploy-aws-s3-cloudfront.js]:
```suggestion
          defaults.ACL= argv.acl;
```

**Comment by jclappiway** [comment]
👍

**Comment by apancutt** [comment]
Please apply the suggestions and this PR will be merged.

**Pull request merged**